backend/src/main/java/com/google/sps/notifiers/EmailNotifier.java

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main.java.com.google.sps.notifiers;
+
+import com.sendgrid.Method;
+import com.sendgrid.Request;
+import com.sendgrid.Response;
+import com.sendgrid.SendGrid;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.*;
+
+import java.io.IOException;
+
+/** Class representing the notification system capable of sending mails to the user. */
+public class EmailNotifier extends Notifier {
+
+  /** Variable representing the email to be notified */
+  private final String toEmail;
+
+  /**
+   * @param name Name of the recipient of the user
+   * @param toEmail Email that this email is going to sent to.
+   */
+  public EmailNotifier(String name, String toEmail) {
+    super(name);
+    this.toEmail = toEmail;
+  }
+
+  /**
+   * Getter function that returns the string representing the email recipients name .
+   *
+   * @return String representing the name of the email recipient.
+   */
+  @Override
```

Comment:
This can use a shorter single-line javadoc format. When javadocs only includes a description of the return type, it can be shortened to a single sentence witohut the @return marker.

/** Returns the String representing the name of the email recipient */

Reply:
Ack